### PR TITLE
Add citations to documentation

### PR DESCRIPTION
**Pull request by FBartos**
Adding citations to the help file -- should be there for the release.

### CODE REVIEW

**Review by AlexanderLyNL — APPROVED** [review]

**Comment by AlexanderLyNL** [comment]
Could you please resolve the conflicts?

**Comment by FBartos** [comment]
done

**Comment by AlexanderLyNL** [comment]
Can you please rebase?

**Comment by FBartos** [comment]
I don't understand why? Didn't I change just a single file in this PR?

**Comment by AlexanderLyNL** [comment]
I don't understand either, but it says 

This branch cannot be rebased due to conflicts

So there's something going on. I guess it relates to the how the conflict was resolved and that the branch is not pointing to right node.

**Comment by FBartos** [comment]
ok, I just did. Did it help?

**Comment by AlexanderLyNL** [comment]
It still says the same thing. It's relatively easy to make a new PR for this. Could you perhaps do so with a checkout from master?

File: inst/help/summarystatsgeneralbayesiantests.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Summary Statistics General Bayesian Tests
 ====================================
 
-The general Bayesian tests allow one to test a hypothesis about a parameter under different data likelihoods.
+The general Bayesian tests allows one to test a hypothesis about a parameter under different data likelihoods and is based on the methodology developed and promoted by Zoltan Dienes (e.g., Dienes 2014; Dienes & Mclatchie, 2018).
```
Comment (AlexanderLyNL) [inline inst/help/summarystatsgeneralbayesiantests.md:4]: By the way, I think that "allow" was fine "and are based" is also better when referring to "The general Bayesian tests"

**Comment by FBartos** [comment]
no idea what's wrong. I created a new [PR](https://github.com/jasp-stats/jaspSummaryStatistics/pull/59) from a different computer instead. Hopefully, it fixes the issue.

**Pull request closed without merging**